tests/layer_tests/mo_python_api_tests/test_mo_convert_pytorch.py

```diff
@@ -178,7 +178,7 @@ def create_pytorch_nn_module_compare_convert_paths_case1(tmp_dir):
     onnx_model_path = os.path.join(tmp_dir, 'export.onnx')
     torch.onnx.export(pt_model, sample_input, onnx_model_path, opset_version=16)
 
-    ref_model = convert_model(onnx_model_path)
+    ref_model = convert_model(onnx_model_path, compress_to_fp16=False)
```

Comment:
Why compress_to_fp16 should be disabled for pytorch?

Reply:
In that case I think we can change the reference graphs for pytorch tests, as it is expected behavior. Probably a separate ticket should be created.

Reply:
Such behavior was expected. I am not sure if a separate ticket should be created but changing reference graph is a solution, and even better if we will have 2 versions of reference one for compressed and one for uncompressed :)

I duplicated this failing tests and added cases where compression is enabled (by default or explicitly) and decompression converts appear in the reference graph.

So, for affected cases we have 1 versions:
1. left reference graph unchanged but specified explicitly `compress_to_fp16=False`
2. added decompression `Convert`s into reference graph and set explicitly `compress_to_fp16=True`
3. added decompression `Convert`s into reference graph but didn't add conversion argument relying on the fact that by default compression is enabled.

This affected `create_pytorch_nn_module_mean_list` and `create_pytorch_nn_module_scale_list`. So now we covered all cases with explicit/implicit compressions. Please review